<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDialog</class>
 <widget class="QDialog" name="LogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>990</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>990</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>990</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyMS - Application File Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/main/iconGTRun.svg</normaloff>:/main/iconGTRun.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QPlainTextEdit" name="txtLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1181</width>
     <height>891</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Consolas&quot;;</string>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PyMS File</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReload">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>950</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>950</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
